File: LeetCode/April Challenge/4th Day/Day_4.cpp
class Solution {
public:
    void moveZeroes(vector<int>& a)
    {
        for(int j=0;j<(count(a.begin(),a.end(),0));j++)
        {    
            for(int i=0;i<(a.size()-1);i++)
                if((a[i]==0)&&(a[i+1]!=0))
                   swap(a[i],a[i+1]);
        }
    }
};

File: LeetCode/April Challenge/4th Day/Day_4.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        new_len = len(nums)
        for i in range(len(nums)):
            if nums[i] == 0:
                new_len -= 1
        for i in range(new_len):
            if nums[i] == 0:
                for j in range(i + 1, len(nums)):
                    if nums[j] != 0:
                        nums[i], nums[j] = nums[j], nums[i]
                        break
        
